swarm/api/client/client_test.go

```diff
@@ -89,8 +89,8 @@ func TestClientUploadDownloadFiles(t *testing.T) {
 		if file.Size != int64(len(expected)) {
 			t.Fatalf("expected downloaded file to be %d bytes, got %d", len(expected), file.Size)
 		}
-		if file.ContentType != file.ContentType {
-			t.Fatalf("expected downloaded file to have type %q, got %q", file.ContentType, file.ContentType)
+		if file.ContentType != "text/plain" {
+			t.Fatalf("expected downloaded file to have type %q, got %q", file.ContentType, "text/plain")
```

Comment:
This should be the other way around (we expect the file to have type "text/plain" but got `file.ContentType`)